internal/graphql/otel_graphql_engine.go

```diff
@@ -0,0 +1,130 @@
+package graphql
+
+import (
+	"context"
+	"sync"
+
+	"go.opentelemetry.io/otel/codes"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/TykTechnologies/graphql-go-tools/pkg/engine/plan"
+	"github.com/TykTechnologies/graphql-go-tools/pkg/engine/resolve"
+	"github.com/TykTechnologies/graphql-go-tools/pkg/graphql"
+	"github.com/TykTechnologies/graphql-go-tools/pkg/operationreport"
+	"github.com/TykTechnologies/graphql-go-tools/pkg/postprocess"
+	"github.com/TykTechnologies/tyk/internal/otel"
+)
+
+type OtelGraphqlEngineV2 struct {
+	mutex          sync.Mutex
+	logger         *logrus.Entry
```

Comment:
Why is the logger needed here?

Reply:
This was leftover code, thank you for spotting